snapcraft.yaml

```diff
@@ -0,0 +1,31 @@
+name: sam-cli
+version: git
+summary: The AWS SAM CLI tool for the AWS Serverless Application Model
+description: >
+  `sam` is the AWS CLI tool for managing Serverless applications written
+  with the AWS Serverless Application Model (SAM).
+  SAM CLI can be used to test functions locally, start a local API Gateway
+  from a SAM template, validate a SAM template, fetch logs, generate sample
+  payloads for various event sources, and generate a SAM project in your
+  favorite Lambda Runtime.
+
+license: Apache-2.0
+
+base: core18
```

Comment:
Does this base image have the correct glibc support? We have had issues building a snap for sam cli because of that?

Has this been tested and verified to work?

Reply:
Yes I have this working locally. I've gone through an `init`, `package`, and `deploy` successfully :)